cirq/google/op_serializer.py

```diff
@@ -68,7 +71,8 @@ def __init__(self,
                  serialized_gate_id: str,
                  args: List[SerializingArg],
                  can_serialize_predicate: Callable[['cirq.Operation'], bool] =
-                 lambda x: True):
+                 lambda x: True,
+                 auto_serialize_tokens: Optional[bool] = True):
```

Comment:
nit: It seems to me "auto" does not add much here; could we call this just `serialize_tokens`? Similarly for the deserializer above, could rename `auto_deserialize_tokens` -> `deserialize_tokens`.

Reply:
Done.